client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -104,10 +108,14 @@ const BrewItem = createClass({
 			</div>
 			<hr />
 			<div className='info'>
-				{brew.tags ? <>
+
+				{brew.tags?.length ? <>
 					<div className='brewTags' title={`Tags:\n${brew.tags.join('\n')}`}>
 						<i className='fas fa-tags'/>
-						{brew.tags.map((tag, idx)=>{return <span>{tag}</span>;})}
+						{brew.tags.map((tag, idx)=>{
+							let matches = tag.match(/^(?:([^:]+):)?([^:]+)$/);
+							return <span className={matches[1]}>{matches[2]}</span>;
```

Comment:
Won't this create spans with empty classnames?

Reply:
No, if `classNames=blank` it doesn't render a class at all.

![image](https://user-images.githubusercontent.com/5878534/188783025-45b7f716-fa85-45fe-ad12-89f6b7595e6a.png)

Reply:
Whew. Good to know react is intelligent.